src/rard/templates/research/partials/section/original_text_details.html

```diff
@@ -18,18 +18,17 @@
         {% endif %}
     </dd>
 
-    <dt class="col-sm-3">{% trans 'Reference' %}</dt>
-    <dd class="col-sm-9">{{ original_text.reference|default:'-' }}</dd>
+    <dt class="col-sm-3">{% trans 'References' %}</dt>
+    <dd class="col-sm-9">
+    {% for i in original_text.reference_list %}{{i}}{% endfor %}
```

Comment:
If reference_list is just a string:
```suggestion
    {{original_text.reference_list}}
```